tests/diceRoller.test.js

```diff
@@ -0,0 +1,58 @@
+const expect = require('chai').expect;
+const assert = require('assert');
+const { rollDice, parseDiceCommand } = require('../commands/diceRoller').forTestsOnly;
+const { register } = require('../commands/diceRoller');
+const { MockBot } = require('./helpers/mockBot');
+const { stripFltSpecialChars } = require('../lib/utils');
+
+describe('diceRoller', function () {
+  describe('#rollDice()', function () {
+    it('should return a number', function () {
+      let results = rollDice({ amount: 1, sides: 4, reroll: true });
+      expect(results).to.be.a('number');
+    });
+    it('should never return a 1 if reroll is true', function () {
+      for (let i = 0; i < 100; i++) {
+        let results = rollDice({ amount: 1, sides: 4, reroll: true });
+        assert(results > 1);
+      }
+    });
+    it('should be able to return a 1 if reroll is false', function () {
+      function testResults() {
+        for (let i = 0; i < 100; i++) {
+          let results = rollDice({ amount: 1, sides: 2, reroll: false });
+          if (results === 1) {
+            return results;
+          }
+        }
+      }
+      assert(testResults() === 1);
+    });
+  });
+  describe('#parseDiceCommand()', function () {
```

Comment:
A useful test to add here would be to parse a complex dice result and make sure that all the values are the right things. Something like

```js
expect(parseDiceCommand('2d6+3r')).toDeepEqual({
  reroll: true,
  ...
});
```